binaries/speed_benchmark_torch.cc

```diff
@@ -129,6 +138,24 @@ int main(int argc, char** argv) {
   torch::jit::GraphOptimizerEnabledGuard no_optimizer_guard(false);
   auto module = torch::jit::load(FLAGS_model);
 
+  if (FLAGS_use_bundled_input >= 0) {
+    auto get_method = module.find_method("get_all_bundled_inputs");
+    if (!get_method) {
+      std::cerr << "Model does not have bundled inputs.  Before saving," << std::endl
+        << "use torch.utils.bundled_inputs.augment_model_with_bundled_inputs." << std::endl;
+      return 1;
+    }
+
+    auto all_inputs = (*get_method)({}).toList();
```

Comment:
Does input bundling do any check for `forward` function signature matching? So that bundled inputs are of the right type? Else it would be nice to add some check/error message here, whenever you can.

Reply:
Yeah.  I don't think the JIT will accept the attribute if it doesn't match the type.